x-pack/legacy/plugins/apm/server/lib/transactions/charts/get_timeseries_data/transform.ts

```diff
@@ -20,8 +20,9 @@ export function timeseriesTransformer({
   bucketSize: number;
 }) {
   const aggs = timeseriesResponse.aggregations;
-  const overallAvgDuration = idx(aggs, _ => _.overall_avg_duration.value);
-  const responseTimeBuckets = idx(aggs, _ => _.response_times.buckets);
+  const overallAvgDuration =
+    idx(aggs, _ => _.overall_avg_duration.value) || null;
+  const responseTimeBuckets = idx(aggs, _ => _.response_times.buckets) || [];
```

Comment:
Why coerce `undefined` to `null` for `overallAvgDuration`?
And why change `responseTimeBuckets`? `getResponseTime` already default to `[]`.

Reply:
In general, values from ElasticSearch are `number | null`, so I thought it would be sensible to stick to that convention. Esp because we have types downstream that expect either `number` or `null`. Aye, will remove the fallback value in this file for `responseTimeBuckets` 👍

Reply:
> In general, values from ElasticSearch are number | null, so I thought it would be sensible to stick to that convention

Good point 👍